GETTING STARTED - Developers.rdoc

```diff
@@ -0,0 +1,61 @@
+==Is this doc for you?
+
+This page offers a Developer's introduction to the Wagn code base and how to customize it for your own needs. it assumes you're a ruby developer or, at least, someone who does not fear ruby code.
+
+It also assumes that you've experimented with Wagn enough to have some familiarity with the basics (cards, types, rules, etc.) <b>If you have no idea what a card is, you may find it helpful to begin at {http://wagn.org Wagn.org}</b> to get a sense of what Wagn's all about.
+
+And, finally, it assumes you're looking at it from a docs site (ie, NOT GitHub).  The page is peppered with links; if they're breaking, try FIXME - (give links here).
+
```

Comment:
This particular document has a pretty clear audience/purpose: it is for folks who are first making the transition from Wagneer to Mod Developer.  It's supposed to give them a high level view of the code and abundant links to dig further into their area of interest.  At the bottom I expect also to include some basic commands for getting started with code.  

Re "It's Cards all the Way Down", to be honest, I never got who or what you were writing that for.  Is there an audience in mind?  Is it a white paper? 

I can see extracting chunks of it to code docs or Wagn.org, but in it's current form I can't personally get through it; I just don't know what I'm reading.  There's no thesis or narrative or explanation as to what you're trying to establish or why.